src/solvers/dg/2d/amr.jl

```diff
@@ -336,7 +336,7 @@ function calc_amr_indicator(dg::Dg2D, mesh::TreeMesh, time)
     base_level = 4
     max_level = 6
     # to make the simulation smaller and quicker wall clock time, choose super_max_level = 6
-    super_max_level = 7
+    super_max_level = 6
```

Comment:
Why did you change it here? Or put differently, why was `7` OK before?

Reply:
For the KHI AMR version, I implemented a special 4-way AMR controller, that allowed a specific super_level = 7 in the core of the vortices...I think this was a bit to playful and might not be a realistic way of doing AMR

Reply:
OK :+1: